src/styles/variables.scss

```diff
@@ -1,3 +0,0 @@
-$material-light: (
-  'background': #f9f9f9
-);
```

Comment:
Hm, I'm surprised this isn't necessary but removing it should improve the application build/rebuild time quite a bit iirc.

Reply:
it was never used!

Reply:
I think this is/was (?) used internally by Vuetify, see this section: https://vuetifyjs.com/en/features/sass-variables/#vue-cli-install

Reply:
I tried reenabling it and changing it to a noxious shade of pink, and it never showed up in the UI. I think we're safe? 🤞

Reply:
Works for me! Faster builds 💪